data/jotl/deck/black-sludge.json

```diff
@@ -129,7 +129,7 @@
           "subActions": [
             {
               "type": "specialTarget",
-              "value": "self"
+              "value": "selfFemale"
```

Comment:
Would like to keep it on "self" and us single label, besides, `selfFemale` is referenced here, in label files it's `self2`.